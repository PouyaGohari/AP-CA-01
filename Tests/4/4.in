3
pa1 pa2 pa3 pa4
pa1:0 YES
pa2:1 NO
pa3:2 IF_NECESSARY|0 YES
pa4:2 NO|1 YES|0 NO