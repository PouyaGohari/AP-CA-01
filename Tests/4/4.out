Time slot 0:
YES: pa1 pa3 
NO: pa4 
IF_NECESSARY: pa2 
###
Time slot 1:
YES: pa4 
NO: pa2 
IF_NECESSARY: pa1 pa3 
###
Time slot 2:
YES: 
NO: pa4 
IF_NECESSARY: pa1 pa2 pa3 
###
